### ISSUE TITLE
Teleporting while in a vehicle causes game to crash or softlock

### ISSUE THREAD

**Issue by Laiedn** [gta-chaos-mod/Trilogy-ASI-Script#235]
Tested in both 2.3.4, my preferred version and the newest 3.2.0 release, and this is happening in both. Whenever a Teleport effect happens, if I'm in a vehicle of any type, the game screen will glitch out completely flashing bright white and a loading screen constantly. Sometimes I'll get lucky and one Teleport will work fine but the next one always causes the issue. I've included a link below to a video of the problem in action.

I found through a bit of testing that this only happens when in a vehicle,  regardless of being on a mission or not. I read through the issues other people had but wasn't able to find this one specificially.

Link to video: https://www.youtube.com/watch?v=bNWDfAWF1O8&ab_channel=DanielWilliams

**Comment by Jersukai** [comment]
I think I had a problem like this at some point. What other mods are you using?

**Comment by Laiedn** [comment]
> I think I had a problem like this at some point. What other mods are you using?

I'm happy you suggested to look at my other mods because I was able to fix it once I did. I was only using SilentPatch and a Widescreen Fix mod, the latter being the problem. I was using the mod that's included in the Rockstar Nexus Downgrader that I use for SA and after removing it, no issues. I instead decided to use the Widescreen Fix from ThirteenAG and I just did a test and again, no issues. So thanks for the suggestion!
